scripts/cli/tasks/closeMilestone.ts

```diff
@@ -0,0 +1,93 @@
+import axios from 'axios';
+import _ from 'lodash';
+import { Task, TaskRunner } from './task';
+
+// const githubGrafanaUrl = 'https://github.com/grafana/grafana';
+
+interface CloseMilestoneOptions {
+  milestone: string;
+}
+
+const closeMilestoneTaskRunner: TaskRunner<CloseMilestoneOptions> = async ({ milestone }) => {
+  const gitHubUsername = process.env.GITHUB_USERNAME;
```

Comment:
Since the close and label removing actions require write permissions we need to pass down credentials. This is the simplest way I could think - unsure if there's a way we currently use:

```bash
$ GITHUB_USERNAME=foo GITHUB_ACCESS_TOKEN=far yarn cli close-milestone -m 12312312
```

Reply:
This is great suggestion and I think both environment variables would make sense to use for all github releated tasks like changelog and cherrypick where we've had some issues with rate limiting. 

Can you please extract creating/getting an axios client to a helper function, with auth information set if environment variables provided? Then update each github related task to get a client instance from that helper method?